.gitignore

```diff
@@ -44,3 +44,5 @@ src/Tools/freecad.zip
 tsupdate_stderr.log
 tsupdate_stdout.log
 files_to_translate.txt
+
+.project
```

Comment:
Why this change?

Reply:
to not get his IDE's config in

Reply:
is a file ignore update for git.
In particular, it does not allow you to commit the eclipse project configuration file